libraries/android/castor-tokens/build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
+    id "com.android.library"
+    id "kotlin-android"
+}
+
+android {
+    def versionFile = file("./version.json")
+    def parsedVersionText = new groovy.json.JsonSlurper().parseText(versionFile.text)
+
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        minSdkVersion 24
```

Comment:
is there any specific reason we set 24 for minSdK?  The [onfido-android-sdk](https://github.com/onfido/onfido-android-sdk)'s minSdkVersion is 21

Reply:
No, this was just all default values when initiating with Android Studio - feel free to identify what and where needs to be changed.